src/lib.rs

```diff
@@ -288,13 +288,6 @@ macro_rules! ImplDekuTraits {
 
                 let input_is_le = endian.is_le();
 
-                if bit_size > max_type_bits {
```

Comment:
This is great! Unfortunately, there's also this case...

```rust
struct DekuTest {
    #[deku(ctx = "deku::ctx::BitSize(9)")]
    field_a: u8
}
```

Reply:
Should be fixed now